README.md

```diff
@@ -59,11 +59,11 @@ Tide variables that users are encouraged to modify begin with the string `tide_`
 
 ## General Variables
 
-| Variable           | Description                                | Default                                      |
-| ------------------ | ------------------------------------------ | -------------------------------------------- |
-| newline            | add empty line before each prompt          | true                                         |
-| left_prompt_items  | order of the left prompt items to display  | pwd git_prompt newline prompt_char           |
-| right_prompt_items | order of the right prompt items to display | status cmd_duration context jobs virtual_env |
+| Variable           | Description                                | Default                                          |
```

Comment:
I should probably have [Prettier](https://github.com/prettier/prettier) as part of the CI workflow, but could you run Prettier on this file to fix these formatting issues? You can install it as a VSCode extension or as a cli command.